### ISSUE TITLE
UI Suggestion - Indicate if there are no comments on a move

### ISSUE THREAD

**Issue by Cogito** [neagle/gokibitz#143]
When browsing a game, when not logged in, there is a prompt to "log in to comment" but no text indicating that there are no comments yet.

It might be nice to reword this to make it more explicit that there are no comments, and that this can be fixed by logging in to add your own!

There is an indication of this on the opening page of the game.

**Comment by neagle** [comment]
![image](https://cloud.githubusercontent.com/assets/176993/21745152/bb09bc9c-d4f2-11e6-8e18-297847d72ef9.png)

Is this what you had in mind?

It's pretty easy to implement, but now I have to decide if I like it. 😄 What problem does this solve for you? It does make it a little clearer that each move has a comment thread, even if it happens to be currently empty. Are you trying to make sure a visitor who drops in at any point knows they could log in and kibitz if they created an account?

**Comment by Cogito** [comment]
Something like that, you could also have text under that with something like:

> Log in to comment

The intent is to try and encourage visitors to add comments.

Another pattern I have seen (not sure if I like this one though) is to allow a comment to be written, but you have to log in to post it. So visitor starts writing, hits submit, and then either logs in or registers.

The idea is to reduce friction, in order to drive engagement.

The reason I raised this in the first place, however, was that the UI changing as you step through a game was jarring. I also found myself jumping around to try and find the 'game-level' comments which were often on the first or last move. The current layout is very clean, but it might be nice to see which moves have comments on them, and be able to jump to those moves quickly. All that probably belongs in another issue though!

**Comment by neagle** [comment]
Thanks for your thoughts on this!

I like keeping instructional text to a minimum where possible and to let UI elements speak for themselves, but this is a case where some of the limitations of that approach are already showing themselves. There actually is a feature to let you jump back and forth between moves with comments on them:

![image](https://cloud.githubusercontent.com/assets/176993/21754794/ba84a99c-d5d5-11e6-8e4c-e61161c75fca.png)

Those left and right arrows will appear on any game with comments, and will navigate between commented moves. And you can even use alt + left and alt + right as keyboard shortcuts to do the same thing. (Try typing ? to get a quick reference for keyboard shortcuts.)

**Comment by Cogito** [comment]
That is very interesting, but yes I didn't notice at all.

I don't think you need instructional text, but what I was thinking was prefixing the comment with the move number it is attached to. Then, under the comment stream list the other moves that have comments.

You already have something very much like this in the form of the 'comment stream' that appears on the game's page.

![image](https://cloud.githubusercontent.com/assets/475164/21754872/1beac554-d65d-11e6-94b6-471d48aad544.png)

The idea would be to condense this, showing just one line per move-with-comment, and showing it on every move.
